RFCs/0007-Editors-API-Specification.md

```diff
@@ -1264,6 +1284,39 @@ This request is used to the end the current test session.
 ```
 
 ## Appendix
+### Setting Target Framework
```

Comment:
Move this below **Key data structures**. Add a shortcut link `<a name="RunSettings.TargetFramework"></a>`. Link it from the test execution request section above.

For an example of linking see how `Message` (in Key Data Structures) is linked.